doc/how_to_guides/ompl_constrained_planning/ompl_constrained_planning.rst

```diff
@@ -18,7 +18,7 @@ The interface currently only supports position constraints on any link of the ro
 
 Configure OMPL
 ^^^^^^^^^^^^^^^^
```

Comment:
Line 11: I don't think this uses the python interface anymore, right? Prob can delete that